jolt-core/src/r1cs/snark.rs

```diff
@@ -74,10 +80,11 @@ impl<F: PrimeField> Circuit<F> for JoltCircuit<F> {
   fn synthesize<CS: ConstraintSystem<F>>(self, cs: &mut CS) -> Result<(), SynthesisError> {
     // TODO(sragss): These paths should not be hardcoded.
     let circuit_dir = JoltPaths::circuit_artifacts_path();
-    let r1cs_path = circuit_dir.join("jolt-fibonacci.r1cs");
-    let wtns_path = circuit_dir.join("jolt-fibonacci_js/jolt-fibonacci.wasm");
+    let r1cs_path = circuit_dir.join("jolt_single_step.r1cs");
```

Comment:
Given these only need to be compiled once, we should add this to the build flow. We can do this in a few places:
- simple build script
- build.rs
- compiler crate

Reply:
Currently: `./jolt-core/src/r1cs/scripts/compile_jolt.sh ./jolt-core/src/r1cs/circuits/jolt_single_step.circom ./target/circuits/`